sdk/src/main/html/hcaptcha.html

```diff
@@ -36,7 +36,9 @@
         window.sysDebug = JSON.parse(window.JSDI.getSysDebug());
     }
 </script>
-<script type="text/javascript">
+<script type="module">
```

Comment:
Needs to be tested on old iOS and Android versions if supported.  @CAMOBAP

Reply:
AFAIK it's part of ES6 standard while async/await is part of ES7.

But in general, it's a good point, we have a reliable way to check supported standard or maybe use babel to "downgrade" ES standart to ES5

Reply:
`jshint` simply skip JavaScript in `<script type="module">...</script>` this is why it wasn't catched by CI

This is what 'breaking changes' was introduced by this PR:

> sdk/build/hcaptcha-form.html: line 40, col 5, 'import' is only available in ES6 (use 'esversion: 6').
> sdk/build/hcaptcha-form.html: line 78, col 5, 'async functions' is only available in ES8 (use 'esversion: 8').
> sdk/build/hcaptcha-form.html: line 176, col 5, 'async functions' is only available in ES8 (use 'esversion: 8').
> sdk/build/hcaptcha-form.html: line 178, col 13, 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).
> sdk/build/hcaptcha-form.html: line 178, col 13, 'destructuring binding' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).
> sdk/build/hcaptcha-form.html: line 199, col 5, 'async functions' is only available in ES8 (use 'esversion: 8').

According to https://caniuse.com/es6 stats (Let me know if you have a better source for such information) not all features sported by Android 4.1 (this is a minimal OS that's supported by hCaptcha SDK)

Action items:
- according to my understanding we may eliminate ES8 features by simply using `Promise` API
- looks like there are no other options and we have to move to ES6

Reply:
We should not exceed ES5 for now.

Reply:
@e271828- AFAIK there is no Promise in ES5 so does it mean that we cannot use `hcaptcha-loader.js` at all?

Reply:
Migrated to index.es5.js